### PR TITLE
Remove modin dependency and add jmcarpenter2 as maintainer

### PR DESCRIPTION
**Pull request by jmcarpenter2**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

* Removes modin dependency, which was causing installation issues and was removed from `swifter >= 1.1.0`
* Hopefully allows adding `swifter >= 1.1.2` to the conda-forge channel

* Adding @jmcarpenter2 as maintainer

### CODE REVIEW

**Comment by jmcarpenter2** [comment]
@conda-forge-admin please rerender

**Comment by jmcarpenter2** [comment]
Hi @conda-forge/core (cc: @BastianZim),

I am trying to update dependencies and add myself as recipe maintainer to the swifter feedstock, but I am new to managing the feedstock. I tried to look through the build logs but it wasn't evident to me as a newcomer what I need to do to resolve the issues. Also worth noting that I just noticed #31 was auto-created and may be causing conflicts with this PR?

No rush, but if you get a chance to advise on how to resolve these failed checks it would be greatly appreciated! Thanks

File: recipe/meta.yaml
Comment (jmcarpenter2) [inline recipe/meta.yaml:10]: updated the sha256 from `openssl sha256 swifter-1.1.2.tar.gz`

File: recipe/meta.yaml
Comment (jmcarpenter2) [inline recipe/meta.yaml]: updated the sha256 from `openssl sha256 swifter-1.1.2.tar.gz`

File: recipe/meta.yaml
Comment (jmcarpenter2) [inline recipe/meta.yaml:1]: i believe this is the only file i updated - i assume the others were auto-updated by conda-forge?

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml]: I only have time tomorrow to look at the rest of the PR but according to the logs you have a SHA mismatch here.

The SHA to use is `b954f3adb894f6b158ae86f4a2bdd44d8d52b75032f997f20a3c8c0b0da58fe4` as can also be seen on https://pypi.org/project/swifter/#copy-hash-modal-a9f3ec52-4764-4ab2-98f9-a8ed9b3eb80a

Comment (jmcarpenter2) [reply recipe/meta.yaml]: Oh wow, thank you so much Bastian - i didnt know you could look at the hash through pypi directly. I really appreciate the hands-on assistance. Just updated the PR and hopefully it passes 🤞

Comment (BastianZim) [reply recipe/meta.yaml]: No worries! For the future, you can also get the SHA by running 
```zsh
curl -sL https://url.com/you/want/to/get.tar.gz | openssl sha256
```

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml:1]: Yes, they got updated when you rerender and should never be changed manually.

**Comment by BastianZim** [comment]
Ok had a sec to look at the recipe and from what I can see it's quite old and doesn't use the newest conda-forge standards so I would recommend updating it completely. A handy tool for that is https://www.marcelotrevisani.com/grayskull or https://github.com/conda-incubator/grayskull which is able to generate a new recipe that should be 95% ready. 
Once you have uploaded that, please rerender using the bot command mentioned above and ping me again and I can have another look.

**Comment by jmcarpenter2** [comment]
> Ok had a sec to look at the recipe and from what I can see it's quite old and doesn't use the newest conda-forge standards so I would recommend updating it completely. A handy tool for that is https://www.marcelotrevisani.com/grayskull or https://github.com/conda-incubator/grayskull which is able to generate a new recipe that should be 95% ready. Once you have uploaded that, please rerender using the bot command mentioned above and ping me again and I can have another look.

Perfect, thanks again @BastianZim! Just updated it and awaiting build success/failure

File: recipe/meta.yaml
Comment (BastianZim) [inline recipe/meta.yaml:46]: Just fyi: Because Grayskull doesn't know about current maintainers, the original maintainer wasn't added so you'll need to add them back manually and rerender afterwards.

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jmcarpenter2) [reply recipe/meta.yaml:46]: ah good catch!

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Let's make sure the last change was captured as well.

File: recipe/meta.yaml
Comment (BastianZim) [inline recipe/meta.yaml]: Please check here about the correct dask package: https://docs.dask.org/en/latest/install.html#conda

File: recipe/meta.yaml
Comment (BastianZim) [inline recipe/meta.yaml:20]: Is 3.6 correct? This is suggested by the conda-forge linter but it can be something else as well obviously.

**Comment by BastianZim** [comment]
Looks great! Just two small questions and this should be good to go!

**Comment by BastianZim** [comment]
Once merged, you can also close all of the other PRs or wait for the bot the close them itself.

File: recipe/meta.yaml
Comment (jmcarpenter2) [reply recipe/meta.yaml:20]: Technically, I believe any Python 3 works, but definitely no support for Python 2

Comment (jmcarpenter2) [reply recipe/meta.yaml:20]: Do people still use python < 3.6?

File: recipe/meta.yaml
Comment (jmcarpenter2) [reply recipe/meta.yaml]: Nice catch! will update

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml:20]: Some systems, yes, but this is up to you as the maintainer. It's perfectly fine to leave it there.

**Comment by jmcarpenter2** [comment]
@conda-forge-admin, please rerender

**Review by apcamargo — APPROVED** [review]

**Comment by BastianZim** [comment]
@apcamargo Would you mind merging this as well? It should be good to go.

**Comment by apcamargo** [comment]
Done! Thanks for the contribution!

**Comment by jmcarpenter2** [comment]
Thanks @apcamargo !

**Comment by apcamargo** [comment]
@jmcarpenter2, if you prefer you can remove me as a maintainer. I recall creating the recipe some years ago because I really wanted swifter in Conda, but I don't think I've ever asked your permission (which is something that conda-forge requires).

**Comment by jmcarpenter2** [comment]
@apcamargo help is always appreciated and I'm 100% confident others have appreciated swifter being on conda, so thank you for the contribution! I'm totally happy for you to continue to be a listed maintainer if you wish to be

**Pull request merged**
